src/tabbar.cpp

```diff
@@ -66,6 +62,12 @@ TabBar::TabBar(QWidget *parent) :
             this, SLOT(onTabMoved(int,int)), Qt::DirectConnection);
 }
 
+void TabBar::openHomePage() {
+    auto current = this->currentWebView();
+    QUITIFNULL(current);
+    current->setUrl("zim://" + current->zimId() + ".zim/");
+}
```

Comment:
This is rather a refactoring change. Maybe it better belong to the code cleanup commit.

Reply:
Agree

Reply:
BTW, do you think this change is beneficial since `openHomePage` is only called in one place?

Reply:
Of course, it is. Now now that place is less cluttered.